crates/cairo-lint-core/tests/test_files/loop_for_while/loop_for_while

```diff
@@ -0,0 +1,181 @@
+//! > loop with arithmetic condition
```

Comment:
can you add one test like that 
```
fn main() {
    let mut x: u16 = 5;
    loop {
        if x * 2 >= 20 {
            break;
        } else {
            x += 1;
        }
    }
}
```
and one like that:
```
fn main() {
    let mut x: u16 = 5;
    loop {
        if x * 2 >= 20 {
            if x > 30 {
                break;
            }
        } else {
            x += 1;
        }
    }
}
```

Reply:
Yeah sure, I will add them asap